src/System.Text.Json/tests/Serialization/OptionsTests.cs

```diff
@@ -25,5 +49,69 @@ public static void DefaultBufferSize()
             options.DefaultBufferSize = 1;
             Assert.Equal(1, options.DefaultBufferSize);
         }
+
+        [Fact]
+        public static void AllowTrailingCommas()
```

Comment:
Can you borrow/add all the allow trailing comma test cases?

https://github.com/dotnet/corefx/blob/ce62a086814fdfecea1d6ba6657c3fb88763e1d6/src/System.Text.Json/tests/Utf8JsonReaderTests.cs#L2576-L2684

Reply:
The tests are just ensuring the matching option is enabled on the reader or writer options, so a simple test is all that is necessary. Otherwise the tests are redundant.